From e96ecd1b55d9502f17bb654e3c99ea58af68a7ad Mon Sep 17 00:00:00 2001
From: Benjamin Kaduk <kaduk@mit.edu>
Date: Wed, 27 Aug 2014 23:55:27 -0400
Subject: Treat krb.hin as a C file for doxygen

Recent releases of doxygen appear to not map unknown extensions to
the C type; since we are processing the configure-input file krb5.hin,
explicitly map it as being a C language file for processing by doxygen.
---
 src/doc/Doxyfile.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/doc/Doxyfile.in b/src/doc/Doxyfile.in
index 2082b6d..c225864 100644
--- a/src/doc/Doxyfile.in
+++ b/src/doc/Doxyfile.in
@@ -4,6 +4,7 @@ JAVADOC_AUTOBRIEF      = YES
 OPTIMIZE_OUTPUT_FOR_C  = YES
 WARN_IF_UNDOCUMENTED   = NO
 SHOW_FILES             = NO
+EXTENSION_MAPPING      = hin=C
 INPUT                  = @SRC@/include/krb5/krb5.hin @DOC@/doxy_examples
 EXAMPLE_PATH           = @DOC@/doxy_examples
 GENERATE_HTML          = NO
